pages/tech-team.tsx

```diff
@@ -0,0 +1,124 @@
+import Layout from "../components/Layout";
+import OfficerJson from "../data/Officer.json";
+import OfficerIcon from "../components/OfficerIcon";
+import TechProject from "../components/TechProject";
+import ProjectsJson from "../data/TechTeamProjects.json";
+
+import { faGithub } from "@fortawesome/free-brands-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+type Props = {
+  techOfficerName: string;
+  techOfficerEmail: string;
+  imgPath: string;
+};
+
+const TeachTeamPage = ({
+  techOfficerEmail,
+  techOfficerName,
+  imgPath,
+}: Props) => (
+  <Layout title="Tech team page">
+    <section className="section">
+      <OfficerIcon
+        name={techOfficerName}
+        position={"Technology Chair"}
+        imgPath={imgPath}
+        email={techOfficerEmail}
+      />
+    </section>
+
+    <section className="section">
+      <div className="container" style={{ textAlign: "center" }}>
+        The technology team is in charge of implementing and maintaining
+        software solutions for the club (including the club website). Members
+        get the opportunity to{" "}
+        <strong>
+          work in a team setting with industry standard technologies
+        </strong>
+        . Very little previous experience is required in order join, as the team
+        is meant to be a learning experience. Outside of our weekly meetings{" "}
+        <strong>(Saturdays 2:00pm-3:00pm CST)</strong>, we require very little
+        time commitment from our members, everyone can work as much or as little
+        as they want.
+      </div>
+
+      <div
+        className="container buttons is-centered"
+        style={{ marginTop: "40px", textAlign: "center" }}
+      >
+        <a className="button" href="https://www.google.com/" target="_blank">
```

Comment:
```suggestion
        <a className="button" href="https://forms.gle/ZGv5Jvk7YYWcgWH76" target="_blank">
```
I guess the interest form is probably the best link for now - I usually don't check it very often unless I had just posted the interest form at a meeting or something like that but I just realized you can get an email notification whenever someone responds so I just enabled that so hopefully that should solve the me not checking the form issue.